lib/fastlane/plugin/wpmreleasetoolkit/actions/common/upload_to_s3.rb

```diff
@@ -143,10 +138,12 @@ def self.available_options
             end,
             optional: true,
             type: Symbol,
-            default_value: nil, # Using nil under the hood until we remove skip_if_exists
+            # We cannot set a default value and have backward compatibility with skip_if_exists at the same time.
+            # (Short of duplicating the default value knowledge in the action implementation)
+            #
+            # We have a test for the default behavior which should hopefully remind us to uncomment this line once well remove skip_if_exists.
+            # default_value: :fail,
```

Comment:
I now that I see and realize that we can't set `default_value: :fail` here like you explain until we get rid of `skip_if_exists` (because otherwise we couldn't handle the fallback mechanism properly), then I think we should probably still keep `default_value: false` for the `skip_if_exists` option until then. Otherwise, printing the doc of the action (`fastlane action upload_to_s3`) will show no default value for any of those two options, making the default behavior undocumented.

So since we can't set `default_value: :fail` on this option yet, at least keeping the default value documented on `skip_if_exists` will inform that the default behavior is for the action to fail.